examples/python/geometry/half_edge_mesh.ipynb

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
<p>this function can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2387/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>